metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
I think there is a difference between packetbeat and metricbeat here. In packetbeat probably most of the time, the response_time is not below 1ms as the requests are happening between two servers. On metricbeat most of the requests should be to localhost and it happens very often, that a request takes less then 1ms. Having lots of 0 inside for response_time would be strange.

I can see `latency` and `request_time` in the context of packetbeat, not sure if it fits metricbeat as we actually "measure" the response time based on a timer.

Summary
* Packetbeat currently has `responsetime`. `latency` or `request_time` would probably be better. If  in ms or ns is not clear to me yet, depends also on Kibana.
* Metricbeat has `response_time`. Ns probably makes sense.

Sharing field names would be good but it seems like we will change Packetbeat in the future so I would suggest to go with `response_time` for metricbeat and "ignore" what we have right now in packetbeat.